### PR TITLE
Update footer

### PR DESCRIPTION
**Pull request by garyfan1**
*Issue #:* /bcgov/entity#12087

*Description of changes:* update the sbc-common-component version to include the new footer, also a change requested by other team member.
image

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

File: pages/dashboard.vue [hunk 1]
```diff
@@ -109,7 +109,7 @@ export default Vue.extend ({
     // fetch data only if we have account id
     if (accountId) {
       // check if user is SBC staff
-      let isSbcStaff: boolean
+      let isSbcStaff = false
```
Comment (cameron-freshworks) [inline pages/dashboard.vue:112]: @severinbeauvais

File: package.json [hunk 2]
```diff
@@ -19,7 +19,7 @@
     "http-status-codes": "^2.1.4",
     "launchdarkly-js-client-sdk": "^2.20.0",
     "nuxt": "^2.15.3",
-    "sbc-common-components": "2.4.40"
+    "sbc-common-components": "2.4.45"
```
Comment (severinbeauvais) [inline package.json:22]: Use the latest (46)...

https://github.com/bcgov/sbc-common-components/releases

Comment (garyfan1) [reply package.json]: oh sorry, updated!

Comment (severinbeauvais) [reply package.json]: Sorry, I don't have permission to resolve this conversation, but it is resolved.

**Review by dimak1 — APPROVED** [review]

**Comment by thorwolpert** [comment]
Temporary Url for review:https://yfthig-dev--pr-74-kks6hqj8.web.app

**Review by severinbeauvais — APPROVED** [review]

**Review by cameron-freshworks — APPROVED** [review]
Looks good!

**Pull request merged**